config/default-admin.js

```diff
@@ -1,4 +1,5 @@
-const amoCDN = 'https://addons.cdn.mozilla.net';
+import { amoProdCDN } from './lib/shared';
+
 const staticHost = 'https://addons-admin.cdn.mozilla.net';
 
 module.exports = {
```

Comment:
Hmm, I don't think there's an easy way to convert all of these to es6 without turning each object key into `export const ...`